airflow/api_fastapi/core_api/routes/public/variables.py

```diff
@@ -86,10 +92,18 @@ def get_variables(
         ),
     ],
     session: SessionDep,
+    filters: list[Filter] = Query(None),
```

Comment:
To remove the assertion you probably need to switch to:
```suggestion
    filters: list[str] | None = Query(None),
```

Reply:
I changes the logic, using more simplified one.